plugins/inputs/openweathermap/openweathermap.go

```diff
@@ -311,7 +321,23 @@ func init() {
 		return &OpenWeatherMap{
```

Comment:
Don't forget to set `Lang` to `"en"` to match the config example.

Reply:
I'll change it to leave out the lang query param if there's no lang setting in the config.  The OWM api already defaults to English if there's no query param.

Reply:
Either way. Seems easier to just set it here in the struct you're returning, vs if not empty logic.